docs/develop.md

```diff
@@ -35,6 +35,8 @@ $ yarn install
 ```sh
 $ yarn start
 ```
+### wirte e2e test case
```

Comment:
md style: need a blank line before and after the title

Reply:
Typo!